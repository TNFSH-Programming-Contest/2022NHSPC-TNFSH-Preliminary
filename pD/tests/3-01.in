8 8
3 1 2 1 1 2 1 3
4 0 2 2 1 1 2 2
...T..T.
..TT....
T.......
.T....T.
....T...
........
T..T.TT.
.T....T.
