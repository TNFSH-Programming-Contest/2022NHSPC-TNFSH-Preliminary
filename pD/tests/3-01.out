..ATA.TA
A.TT....
T..A..A.
AT....T.
...AT...
A.....A.
T.AT.TT.
AT...ATA
